variables.tf

```diff
@@ -348,7 +348,7 @@ variable "datadog_metrics_waggledance" {
   ]
 }
 
-variable "datadog_metrics_port" {
+variable "metrics_port" {
```

Comment:
this variable is not in README. Please regenerate the markdown table using the terraform-docs command.